www/class/centreon-clapi/centreon.Config.Poller.class.php

```diff
@@ -543,10 +545,66 @@ public function cfgMove($variables)
                 $msg_copy .= _("OK: All configuration files copied with success.");
             }
         } else {
-            exec("echo 'SENDCFGFILE:" . $host['id'] . "' >> " . $this->centcore_pipe, $stdout, $return);
-            if (!isset($msg_copy)) {
-                $msg_copy = "";
+            /**
+             * Get Parent Remote Servers of the Poller
+             */
+            $statementRemotes = $pearDB->prepare(
+                'SELECT ns.id
+                FROM nagios_server AS ns
+                JOIN platform_topology AS pt ON (ns.id = pt.server_id)
+                WHERE ns.id = :pollerId
+                AND pt.type = "remote"
+                UNION
+                SELECT ns1.id
+                FROM nagios_server AS ns1
+                JOIN platform_topology AS pt ON (ns1.id = pt.server_id)
+                JOIN nagios_server AS ns2 ON ns1.id = ns2.remote_id
+                WHERE ns2.id = :pollerId
+                AND pt.type = "remote"
+                UNION
+                SELECT ns1.id
+                FROM nagios_server AS ns1
+                JOIN platform_topology AS pt ON (ns1.id = pt.server_id)
+                JOIN rs_poller_relation AS rspr ON rspr.remote_server_id = ns1.id
+                WHERE rspr.poller_server_id = :pollerId
+                AND pt.type = "remote"'
+            );
+            $statementRemotes->bindValue(':pollerId', $pollerId, \PDO::PARAM_INT);
+            $statementRemotes->execute();
+            $remotesResults = $statementRemotes->fetchAll(\PDO::FETCH_ASSOC);
+
+            /**
+             * If the poller is linked to one or many remotes
+             */
+            foreach ($remotesResults as $remote) {
+                $linkedStatement = $pearDB->prepare(
+                    'SELECT id
+                    FROM nagios_server
+                    WHERE remote_id = :remoteId
+                    UNION
+                    SELECT poller_server_id AS id
+                    FROM rs_poller_relation
+                    WHERE remote_server_id = :remoteId'
+                );
+                $linkedStatement->bindValue(':remoteId', $remote['id'], \PDO::PARAM_INT);
+                $linkedStatement->execute();
+                $linkedResults = $linkedStatement->fetchAll(\PDO::FETCH_ASSOC);
+
+                $exportParams = [
+                    'server' => $remote['id'],
+                    'pollers' => []
+                ];
+
+                if (!empty($linkedResults)) {
+                    $exportParams['pollers'] = array_column($linkedResults, 'id');
+                } else {
+                    $exportParams['pollers'] = [$remote['id']];
+                }
+
+                $dependencyInjector['centreon.taskservice']->addTask(Task::TYPE_EXPORT, ['params' => $exportParams]);
```

Comment:
use constant for `centreon.taskservice`  : https://github.com/centreon/centreon/blob/master/src/CentreonRemote/ServiceProvider.php#L44